packages/docs/docs/tutorials/importing-sample-data.md

```diff
@@ -0,0 +1,61 @@
+| Smoking Status                                                                                                                                                                                                                                                                                                                                                                              | [Smoking Status Observation](https://app.medplum.com/Observation?_count=20&_fields=_lastUpdated,subject,code,category,status,effective[x],issued&_offset=0&_sort=-_lastUpdated&code:contains=tobacco)                                                                                                                                                  | [Observation](/docs/api/fhir/resources/observation)                                                                                                                                          |
+| Implantable Device                                                                                                                                                                                                                                                                                                                                                                          | [Device](https://app.medplum.com/Device?_count=20&_fields=_lastUpdated,manufacturer,deviceName,patient&_offset=0&_sort=-_lastUpdated)                                                                                                                                                                                                                  | [Analytics](/docs/analytics)                                                                                                                                                                 |
+| Vitals: <br /> ● Diastolic blood pressure <br /> ● Systolic blood pressure <br /> ● Body height <br /> ● Body weight <br /> ● Heart rate <br /> ● Respiratory rate <br /> ● Body temperature <br /> ● Pulse oximetry <br /> ● Inhaled oxygen concentration <br /> ● BMI Percentile (ages 2-20) <br /> ● Weight-for-length (0-36mos) <br /> ● Occipital-frontal Head Circumference (0-36mos) | [Vital Specific Observation](https://app.medplum.com/Observation?_count=20&_fields=_lastUpdated,subject,code,category,status,effective[x],issued&_offset=0&_sort=-_lastUpdated&category:contains=Vital-Signs)                                                                                                                                          | [Observation](/docs/api/fhir/resources/observation)                                                                                                                                          |
+
+We recommend looking through the tabs on resource page (e.g. Edit, History, JSON) to see how the data is represented. Below is a sample of what the tabs look like in the Medplum app.
```

Comment:
If you make this the 2nd tutorial, I would break this up into a new section (something like "Explore the Data") and give a more guided tour of how to view the resource. Assume no familiarity with the Medplum app